### DISCUSSION TITLE
Something about imagedata check when start labelme

### DISCUSSION THREAD

**Discussion by zhuang97** [wkentaro/labelme#1086] [Q&A / Help] [answered]
My problem:
In some cases, I need to change the image size, format and others about the image after the image is marked. After converting the image, I changed the value of the corresponding items in the json file except imagedata to match the new image. However, an error will be reported when using labelme to open it, which indicating that the width and height do not match the imagedata.

My desire:
Is there a method that labelme load image without checking imagedata

**Comment by wkentaro** [comment, marked as answer]
I guess a solution would be edit the JSON files and remove `"imageData"` sections.
